card/lib/generators/card/USAGE

```diff
@@ -0,0 +1,21 @@
+Description
+
+Examples
+  wagn generate card new cards pre_fix
```

Comment:
I just added a commit to this branch that improves things.  This is the basic scenario as I see it:

1. You have a rails app (generate a skeleton app for testing).
2. Add decko-rails (later just decko) to the Gemfile
3. Generate a migration to add cards to my app: rails generate card [optional args, defaults are just like a deck app]
4. Follow the instructions output from step 3.
5. Edit config/routes.rb, config/application.rb and Rakefile
6. Run migrations: rake db:migrate
7. Load card seeds: rake decko:seed
8. Run tests: rake decko:rspec (this doesn't exist yet, but I think this is what we want for starters)

We could make it so wagn (later decko) command will run the tests for either case, but right now decko doesn't exist and wagn rspec won't run in an engine app.  We could fix it so it does and eventually port that to decko when we factor out wagn.